apps/console/src/features/connections/resources/guides/swe/quick-start.tsx

```diff
@@ -210,33 +211,20 @@ const SIWEAuthenticationProviderQuickStart: FunctionComponent<SIWEAuthentication
                                 ".content"
                             }
                         >
-                            Go to <strong>Login Flow</strong> tab and click on <strong>Start with default
-                            configuration</strong>.
+                            Go to <strong>Login Flow</strong> tab and click on the <strong>Add Sign In Option</strong>
+                            button inside the login box. And select a Sign In With Ethereum connection.
                         </Trans>
                     </Text>
-                    <GenericIcon inline transparent icon={ BuildLoginFlowIllustration } size="huge"/>
+                    <GenericIcon inline transparent icon={ BuildLoginFlowStep01Illustration } size="huge"/>
+                    <GenericIcon inline transparent icon={ BuildLoginFlowStep02Illustration } size="huge"/>
+                    <GenericIcon inline transparent icon={ BuildLoginFlowStep03Illustration } size="huge"/>
                 </>
             ),
             stepTitle: (
                 <Trans i18nKey="extensions:develop.identityProviders.siwe.quickStart.steps.selectDefaultConfig.heading">
-                    Select <strong>Start with default configuration</strong>
+                    Add a <strong>Sign In With Ethereum</strong> connection
```

Comment:
Let's verify this as well

Reply:
Refer: https://github.com/wso2/identity-apps/pull/5551#discussion_r1492035610